app/code/Magento/InventoryApi/Api/SourceItemRepositoryInterface.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\InventoryApi\Api;
+
+interface SourceItemRepositoryInterface
+{
+    /**
+     * Save Source item data. If you want to create plugin on get method, also you need to create separate plugin
+     * on getList method, because entity loading way is different for these methods
+     *
+     * @param \Magento\InventoryApi\Api\Data\SourceItemInterface $sourceItem
+     * @return int
+     * @throws \Magento\Framework\Exception\CouldNotSaveException
+     */
+    public function save(\Magento\InventoryApi\Api\Data\SourceItemInterface $sourceItem);
```

Comment:
but it's not clear how to handle multiple statuses, we don't have dedicated interfaces which return statuses for Bulk API.
Like source item IDs 1,2,3,5 saved sucessfully
4 and 6 are failed to save.

Will raise this question on Architectural meeting